services/udp/service.go

```diff
@@ -18,10 +18,6 @@ import (
 )
 
 const (
-	// UDPBufferSize is the maximum UDP packet size
-	// see https://en.wikipedia.org/wiki/User_Datagram_Protocol#Packet_structure
-	UDPBufferSize = 65536
```

Comment:
Can you rename this constant to `MaxUDPBufferSize` and modify the service's `Open` function to return an error when  the payload size is set too high in the config?

It would be nice to either error or give some kind of warning to the user if they set the buffer size unnecessarily high.